src/frontend/context/ProjectInviteContext.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+
+export interface ProjectInvite {
+  project: {
+    id: string;
+    name: string;
+  };
+  role: string;
+}
+
+const ProjectInviteContext = React.createContext<{
+  invite?: ProjectInvite;
+  removeInvite: (inviteId: string) => void;
+}>({ removeInvite: () => null });
+
+export const ProjectInviteProvider = ({
+  children,
+}: React.PropsWithChildren<{}>) => {
+  const [invites, setInvites] = React.useState<ProjectInvite[]>([]);
+
+  // TODO: Need to integrate backend/interactions that add invite
+
+  // TODO: Need to incorporate disabling based on current route (needs coordination with useProjectInviteListener)
```

Comment:
highlighting this todo since it's something I figured we can work on as a follow-up